components/sbm-recipes-mule-to-boot/src/main/java/org/springframework/sbm/mule/actions/javadsl/translators/dwl/DwlTransformTranslator.java

```diff
@@ -60,13 +91,19 @@ public DslSnippet translate(
             String flowName,
             Map<Class, MuleComponentToSpringIntegrationDslTranslator> translatorsMap
     ) {
+        // Ugly hack to work around an inability to inject a sbm property into the mulesoft parser.
+        String isTmTransformationEnabled = System.getProperty("sbm.muleTriggerMeshTransformEnabled");
```

Comment:
Required to work around case when running unit test and `@Autowired` is unable to bring in the SBM properties.